File: SEM-5/APP/palindrome.R
n <- as.integer(readline(prompt="Enter a Number : "))
num <- n
rev <- 0L
while(n != 0){
  rem = n %% 10
  rev = rem + (rev*10)
  n = as.integer(n/10)
}
if(num == rev){
  cat(num,"is a Palindrome number !")  
}else{
  cat(num,"is not a Palindrome number !")
}

File: SEM-5/JAVA/Palindrome.java
import java.util.*;

class Palindrome
{
	public static void main(String[] args) 
	{
		Scanner s = new Scanner(System.in);
		int num,temp,rem;
		int sum=0;

		System.out.println("Enter a Number to check weather it is palindrome or not");
		num = s.nextInt();
		temp = num;

		while(num > 0)
		{
			rem = num%10;
			sum = (sum*10)+rem;
			num = num/10;

		}

		if(temp == sum)
		{
			System.out.println("Entered number is a Palindrome Number.");
		}
		else
		{
			System.out.println("Entered number is not a Palindrome Number.");
		}
	}
}